resources/web/wwi/Makefile

```diff
@@ -34,9 +34,34 @@ JS_SOURCES = \
   animation.js \
   webots.js
 
-release debug profile update: $(JS_SOURCES)
-	@echo "# Compressing webots.js.min"
-	@terser $(JS_SOURCES) -o webots.min.js
+NPM_PACKAGES = \
+  babel-minify \
+  @babel/core \
+  @babel/cli \
+  @babel/preset-env
+
+NPM_EXISTS = $(shell which npm 2> /dev/null)
+
+ifeq ($(NPM_EXISTS),)
+release debug profile update:
+	@echo "# Please install nodejs to build webots.min.js..."
+	@echo "- Linux: sudo apt install npm"
+	@echo "- macOS: brew install node"
+	@echo "- Windows: pacman -S mingw-w64-x86_64-nodejs"
+else
+release debug profile update: install $(JS_SOURCES)
```

Comment:
This Makefile when invoked twice in a row seems to repeat useless operations and is very verbose (tested on Windows):
```
$ make
npm install babel-minify @babel/core @babel/cli @babel/preset-env
npm WARN saveError ENOENT: no such file or directory, open 'D:\msys64\home\Olivier\webots\resources\web\wwi\package.json'
npm WARN enoent ENOENT: no such file or directory, open 'D:\msys64\home\Olivier\webots\resources\web\wwi\package.json'
npm WARN wwi No description
npm WARN wwi No repository field.
npm WARN wwi No README data
npm WARN wwi No license field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

+ babel-minify@0.5.0
+ @babel/cli@7.4.3
+ @babel/core@7.4.3
+ @babel/preset-env@7.4.3
updated 4 packages in 2.673s
# Compressing webots.min.js
```